bio/deeptools/bampefragmentsize/wrapper.py

```
@@ -0,0 +1,51 @@
+
+out_file = snakemake.output.get("hist")
+
+# Check output format
+out_format = out_file.split(".")[-1]
+if not out_format in ["png", "pdf", "svg", "eps", "plotly"]:
+    raise ValueError(
+        "Output format must be either 'png', 'pdf', 'svg', 'eps', or 'plotly'"
+    )
+
+# Optional output
+out_raw = snakemake.output.get("raw", "")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)